cmd/aries-js-worker/main.go

```diff
@@ -94,7 +98,7 @@ func takeFrom(in chan *command) func(js.Value, []js.Value) interface{} {
 }
 
 func pipe(input chan *command, output chan *result) {
-	handlers := defaultHandlers()
+	handlers := testHandlers()
```

Comment:
testHandlers?

Reply:
in my previous PR @llorllale asked to leave some test handlers behind to test JS wasm integration. May be time to take them off now?

Reply:
No, not yet